src/App/redirects/AlertDetailsRedirect.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+    generatePath,
+    useNavigate,
+    useParams,
+} from 'react-router-dom';
+
+import routes from '#routes';
+
+interface AlertDetailsParams {
+    alertId: string | undefined;
+    [key: string]: string | undefined;
```

Comment:
Do we need this?

Reply:
We need to pass  Index signature for useParams.